src/pull-request-handler.ts

```diff
@@ -55,11 +57,11 @@ export type PullRequestAction = 'reschedule' | 'update_branch' | 'merge' | 'dele
 export type PullRequestActions
   = (
     []
-  | ['reschedule']
-  | ['update_branch', 'reschedule']
-  | ['merge']
-  | ['merge', 'delete_branch']
-) & Array<PullRequestAction>
+    | ['reschedule']
+    | ['update_branch', 'reschedule']
+    | ['merge']
+    | ['merge', 'delete_branch']
+  ) & Array<PullRequestAction>
```

Comment:
Does the linter think differently about this?

Reply:
This is what I get when running `npm run lint`.